.travis.yml

```diff
@@ -31,16 +30,19 @@ install:
                :prompt nil)'
 
 script:
-  - ros -e '(ql:quickload :emotiq-test) 
-            (when (asdf:test-system :emotiq)
-               (uiop:quit 0))'
-  - ros -e '(ql:quickload :cosi-test)
-            (when (asdf:test-system :cosi)
-               (uiop:quit 0))'
-  # Speculatively hook into Coveralls <https://coveralls.io/>
-  # Coveralls invocation (currently unused)
-  #                 
-  # - ros -s prove -s cl-coveralls
-  #       -e '(or (coveralls:with-coveralls (:exclude (list "t"))
-  #                 (prove:run :emotiq-prove))
-  #               (uiop:quit -1))'
+  - ros -e "(loop
+              :for i :upfrom 0
+              :for 
+                  (system dependencies) 
+              :in 
+                  '((:emotiq :emotiq-test)
+                    (:cosi   :cosi-test))
+              :doing 
+                 (ql:quickload dependencies)
+              :unless 
+                 (asdf:test-system system)
+              :return 
+                 (uiop:quit (ash -1 i))
```

Comment:
how about quit with i

Reply:
The `ash` forms gives different exit status depending on which test has failed.

Reply:
Don't think it's useful information to shift bits for each test number. Also, can get into bignums, may not work well for interprocess. Also, don't see use for negative numbers Suggest just returning test index. Has to be non-zero, so can just return (1+ i) here....